modules/install-sno-ibm-z.adoc

```diff
@@ -150,20 +150,20 @@ The `rootfs` image is the same for FCP and DASD.
 +
 [source,terminal]
 ----
-rd.neednet=1 \
+cio_ignore=all,!condev rd.neednet=1 \
 console=ttysclp0 \
-coreos.live.rootfs_url=<rhcos_liveos>:8080/rootfs.img \// <1>
 ignition.firstboot ignition.platform.id=metal \
-ignition.config.url=<rhcos_ign>:8080/ignition/bootstrap-in-place-for-live-iso.ign \// <2>
-ip=encbdd0:dhcp::02:00:00:02:34:02 <3>
+ignition.config.url=http://<http_server>:8080/ignition/bootstrap-in-place-for-live-iso.ign \// <1>
+coreos.live.rootfs_url=http://<http_server>/rhcos-<version>-live-rootfs.<architecture>.img \// <2>
+ip=<ip>::<gateway>:<mask>:<hostname>::none <3>
 rd.znet=qeth,0.0.bdd0,0.0.bdd1,0.0.bdd2,layer2=1 \
 rd.dasd=0.0.4411 \// <4>
 rd.zfcp=0.0.8001,0x50050763040051e3,0x4000406300000000 \// <5>
 zfcp.allow_lun_scan=0 \
 rd.luks.options=discard
```

Comment:
Why do we need this parameter ?

Reply:
@chidanandbh Can you pleases take a look. Original input came from you when we added support for SNO